scalafix-tests/unit/src/test/scala/scalafix/tests/cli/ScalafixImplSuite.scala

```diff
@@ -28,7 +28,7 @@ import scalafix.internal.reflect.RuleCompilerClasspath
 import scalafix.test.StringFS
 import scalafix.testkit.DiffAssertions
 import scalafix.tests.util.ScalaVersions
-import scalafix.tests.util.SemanticdbPlugin
+import scalafix.tests.util.compat.SemanticdbPlugin
```

Comment:
here we're also calling nsc.Main.process @ line 195:
`    val compileSucceeded = scala.tools.nsc.Main.process(scalacOptions)
`

Reply:
then you'll probably need to update the `scalacOptions` as well